assets/src/hooks/useDetour.ts

```diff
@@ -88,7 +88,11 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
   )
 
   const coordinates = isOk(detourShape) ? detourShape.ok.coordinates : []
-  const directions = isOk(detourShape) ? detourShape.ok.directions : undefined
+  const directions = isOk(detourShape)
+    ? detourShape.ok.directions?.concat({
+        instruction: "Return to Regular Route",
```

Comment:
question: this text doesn't match the copy in the Figma, was it updated somewhere else that I missed?

Reply:
Ohhhhh!! I got so locked onto the ticket title that I glossed over the diff with the figma